src/plugins/mmapstorage/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+#[[
```

Comment:
I've disabled the build of `mmapstorage` for now, because obviously the plugin does not compile at this time.